frontend/src/containers/Init/Init.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+
+import { Form, Button, ButtonGroup } from "react-bootstrap";
+import { authActions } from "../../store/actions";
+
+import "./Init.css";
+
+class Init extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            keywords: [],
+            checkedKeywords: [],
+        };
+
+        this.checkHandler = this.checkHandler.bind(this);
+        this.clickMoreHandler = this.clickMoreHandler.bind(this);
+        this.clickConfirmHandler = this.clickConfirmHandler.bind(this);
+    }
+
+    componentDidMount() {
```

Comment:
I think we have some codes redirecting users accessing this page inappropriately.

Reply:
Good